webroot/src/pages/SelectPrivileges/SelectPrivileges.less

```diff
@@ -0,0 +1,198 @@
+            width: 100%;
+
+            .right-cell {
+                display: flex;
+                flex-direction: row;
+                align-items: flex-end;
+                justify-content: space-between;
+
+                .back-button {
+                    margin-right: 1rem;
+                }
+            }
+        }
+
+        .select-privileges-core-container {
+            margin-top: -1rem;
+            padding: 2rem;
+            border-radius: 20px;
+            background-color: white;
+
+            @media @tabletWidth {
+                width: 72rem;
+            }
+
+            @media @desktopWidth {
+                width: unset;
+                min-width: 72rem;
+                margin-top: 8rem;
+            }
+
+            .select-privileges-title {
+                margin: 0 1rem 2rem 1rem;
+                font-size: @fontSizeLarger;
+            }
+
+            .lists-container {
+                display: flex;
+                flex-direction: column;
+                align-items: center;
+
+                @media @tabletWidth {
+                    flex-direction: row;
+                    align-items: flex-start;
+                    justify-content: space-between;
+                }
+
+                .state-select-list {
+                    display: flex;
+                    flex-flow: column wrap;
+
+                    @media @desktopWidth {
+                        max-height: 65vh;
+                    }
+
+                    .state-select-unit {
+                        width: 27rem;
+                        margin: 1rem;
+                        padding: 0 1rem;
+                        border: 1px solid @lightGrey;
+                        border-radius: 10px;
+
+                        :deep(.checkbox-container) {
+                            label {
+                                padding: 0 5rem 0 0.4rem;
+                                font-weight: @fontWeight;
+                            }
+                        }
+
+                        .disabled-state-overlay {
+                            position: absolute;
+                            z-index: 2;
+                            width: 100%;
+                            height: 100%;
+                            margin: 0 -1rem;
+                            border-radius: 10px;
+                            background-color: @lightGrey;
+                            opacity: 0.5;
+                        }
+                    }
+                }
+
+                .selected-state-list {
+                    display: flex;
+                    flex-direction: column;
+                    align-items: center;
+
+                    .selected-state-block {
+                        width: 27rem;
+                        margin: 1rem 0;
+                        padding: 1rem 1.6rem;
+                        border-radius: 20px;
+                        color: white;
+                        background-color: @darkBlue;
```

Comment:
This might be easier as `@primaryColor` for now (which is black). That's what I've been using in anticipation that primary color will change globally (and hopefully easily) once we implement the hi-fi designs.

Reply:
Resolved.